src/pages/DrowsinessSearch.jsx

```
@@ -7,144 +7,215 @@ import 'react-datepicker/dist/react-datepicker.css';
+    },
+    {
+      uid: 102,
+      vehicleNumber: '34나5678',
+      deviceUid: 'rasp-0002',
+      detectedTime: '2025-05-16T15:10:00',
+      driverHash: 'driver-bbb-222',
+      videoPath: '/videos/sleep_102.mp4',
+    },
+  ];*/
+  }
+  const [selectedDriverIndex, setSelectedDriverIndex] = useState(null);
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)